jobs/update-colin-filings/config.py

```diff
@@ -52,19 +52,21 @@ class _Config:  # pylint: disable=too-few-public-methods
 
     # API Endpoints
     BUSINESS_API_URL = os.getenv("BUSINESS_API_URL", "")
-    BUSINESS_API_VERSION_2 = os.getenv("BUSINESS_API_VERSION2", "")
+    BUSINESS_API_VERSION_2 = os.getenv("BUSINESS_API_VERSION_2", "")
     COLIN_API_URL = os.getenv("COLIN_API_URL", "")
     COLIN_API_VERSION = os.getenv("COLIN_API_VERSION", "")
 
-    COLIN_API = f"{COLIN_API_URL + COLIN_API_VERSION}"
-    LEGAL_URL = f"{BUSINESS_API_URL + BUSINESS_API_VERSION_2}"
+    COLIN_URL = f"{COLIN_API_URL + COLIN_API_VERSION}/businesses"
+    LEGAL_URL = f"{BUSINESS_API_URL + BUSINESS_API_VERSION_2}/businesses"
 
     # service accounts
     ACCOUNT_SVC_AUTH_URL = os.getenv("KEYCLOAK_AUTH_TOKEN_URL")
     ACCOUNT_SVC_CLIENT_ID = os.getenv("KEYCLOAK_CLIENT_ID")
-    ACCOUNT_SVC_CLIENT_SECRET = os.getenv("KEYCLOAK_CLIENT_ID")
+    ACCOUNT_SVC_CLIENT_SECRET = os.getenv("KEYCLOAK_CLIENT_SECRET")
```

Comment:
I've encountered the same thing in legal-api: https://github.com/bcgov/lear/commit/9b730bdf6fd8c8761e058300461b156e23ccd3a4

I wonder if there's more of this some place else.

Reply:
yeah, i'm sure there are.  The future work for the various BE services should uncover issues like this.  The GCP integration check tickets are there too.